README.md

```diff
@@ -34,241 +30,10 @@ If you already cloned and just need to get the submodules, you can run:
-
-To force reindexing of a task's data, delete some or all of the objects in `preproc/` or use the option ``reload_index = 1`` and set ``reindex_tasks`` to the names of the tasks to be reindexed, e.g. ``reindex_tasks=\"sst,mnli\"``. You should do this whenever you rebuild the task objects or vocabularies.
-
-## Models (and how to add a Model)
-
-The core model is a shared BiLSTM with task-specific components. When a language modeling objective is included in the set of training tasks, we use a bidirectional language model for all tasks, which is constructed to avoid cheating on the language modeling tasks. We also provide bag of words and RNN sentence encoder.
-
-The base model class is a MultiTaskModel. To add another model, first add the class of the model to modules/modules.py, and then add the model construction in ``make_sent_encoder()`` (called in ``build_model()``) in src/models.py.
-
-Task-specific components include logistic regression and multi-layer perceptron for classification and regression tasks, and an RNN decoder with attention for sequence transduction tasks.
-To see the full set of available params, see [config/defaults.conf](config/defaults.conf). For a list of options affecting the execution pipeline (which configuration file to use, whether to enable remote logging or tensorboard, etc.), see the arguments section in [main.py](main.py).
-
-To use the ON-LSTM sentence encoder from [Ordered Neurons: Integrating Tree Structures into Recurrent Neural Networks](https://arxiv.org/abs/1810.09536), set ``sent_enc = onlstm``. To re-run experiments from the paper on WSJ Language Modeling, use the configuration file [config/onlstm.conf](config/onlstm.conf). Specific ON-LSTM modules use code from the [Github](https://github.com/yikangshen/Ordered-Neurons) implementation of the paper.
-
-To use the PRPN sentence encoder from [***Neural language modeling by jointly learning syntax and lexicon***](https://arxiv.org/abs/1711.02013), set ``sent_enc=prpn``. To re-run experiments from the paper on WSJ Language Modeling, use the configuration file [config/prpn.conf](config/prpn.conf). Specific PRPN modules use code from the [Github](https://github.com/yikangshen/PRPN) implementation of the paper. 
-
-## Currently Supported Task Types
-
-We currently support the following:
-
-	* Single sentence classification tasks
-	* Pair sentence classification tasks
-	* Regression tasks
-	* Tagging tasks
-	* Span classification Tasks 
-		* To run these, we currently require an extra preprocessing step, which consists of preprocessing the data to get BERT tokenized span indices. SpanTasks expects the files to be in `json` format and be named as `{file_name}.retokenized.{tokenizer_name}`.
-	* seq2seq tasks (partial/tentative support only) partially supported.
-
-### GPT, BERT, and Transformers 
```

Comment:
Probably best to keep in the readme?

Reply:
My concern is that the README will get too long (as opposed to having it in a documentation site).